tests/integ/test_tf_script_mode.py

```diff
@@ -137,6 +139,10 @@ def test_mnist_async(sagemaker_session):
 
         result = predictor.predict(np.zeros(784))
         print('predict result: {}'.format(result))
+        _assert_endpoint_tags_match(sagemaker_session.sagemaker_client, predictor.endpoint, TAGS)
+        _assert_model_tags_match(sagemaker_session.sagemaker_client, estimator.latest_training_job.name, TAGS)
+
+    _assert_training_job_tags_match(sagemaker_session.sagemaker_client, estimator.latest_training_job.name, TAGS)
```

Comment:
move this up above with the training part of the test

Reply:
will do